galaxy.yml

```
@@ -0,0 +1,35 @@
+namespace: 'hifis'
+name: 'software_services'
+version: '4.0.0'
+readme: 'README.md'
+authors:
+- 'HIFIS Software Services <support@hifis.net>'
+description: 'This collection provides production-ready Ansible roles used for providing services in the field of research software.'
```

Personally, I think this is not precise and misleading for Ansible developers, because Ansible developers would ask themselves, why should they use this role if it is specific to research software. This rather explains where HIFIS comes from. Our roles and this collection is more general and applicable to more cases but research software. 

```suggestion
description: 'This collection provides production-ready Ansible roles used for providing services used in research and by research software engineers, but not exclusively.'
```

This might be nitpicking, but I would be confused about this description, that's why I suggest to find a more general description. Any ideas?

---

You're right, your version is slightly better. :+1: